src/app/lib/logger.web.js

```diff
@@ -2,12 +2,12 @@
 
 class Logger {
   constructor() {
-    this.error = arg => console.error(`error - ${arg}`);
-    this.warn = arg => console.warn(`warn - ${arg}`);
-    this.info = arg => console.info(`info - ${arg}`);
-    this.verbose = arg => console.log(`verbose - ${arg}`);
-    this.debug = arg => console.debug(`debug - ${arg}`);
-    this.silly = arg => console.log(`silly - ${arg}`);
+    this.error = (event, message) => console.error({ event, message });
+    this.warn = () => () => {};
+    this.info = () => () => {};
+    this.verbose = () => () => {};
+    this.debug = () => () => {};
+    this.silly = () => () => {};
```

Comment:
Thanks for these changes, really appreciate it.

A bit of a tweak to this behaviour but could we consider still logging to the console in a development build but not a in prod? 

You can use `process.env.NODE_ENV === 'production'` to differentiate between the dev and prod. You can set this value in the tests to verify the different behaviour.